addons/minedetector/functions/fnc_playDetectorSound.sqf

```diff
@@ -27,6 +26,9 @@ if (!alive _unit) exitWith {
     ACE_LOGERROR_1("unit is not alive [%1]",_unit);
 };
 
-private _posASL = AGLtoASL (_unit modelToWorld (_unit selectionPosition "granat"));
-
-playSound3D [_sound, objNull, false, _posASL, 5, _pitch, 15];
+if (_unit getVariable [QGVAR(isUsingHeadphones), false]) then {
+    _unit say2D _soundClass;
```

Comment:
Ok, I'll be sure to test if queuing happens or not with playSound, I think it doens't, cause we already use it on many places in ACE without problems (like combat deafness).